enhancements/authentication/login-logout-events.md

```diff
@@ -102,6 +102,92 @@ const (
 )
 ```
 
+We propose to add additional information that helps to identify login and login failures to the audit log.
+
+In case that the authentication happens through the `oauth-server`, we suggest to add:
+
+- `authentication.openshift.io/username`, which is the username for the authentication attempt.
+- `authentication.openshift.io/authn-method`, which mentions the OAuth2 authentication method in use.
+- `authentication.openshift.io/decision`, which is an enum that can be `allow` or `deny`.
+- `authentication.openshift.io/reason`, which defines a reason in case that the `authentication.openshift.io/decision` is set to `deny`. In case that it is `allow`, the value is an empty `string`.
```

Comment:
Aren't these two going to be redundant to the response status and message?

Reply:
More or less... the reason might have more context.